exo-sync/bin/setup

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-npm install --loglevel error --depth 0
+yarn install
```

Comment:
Check whether yarn is installed here, and if not install it by running `npm i -g yarn`?